v1.0/Specification.md

```diff
@@ -342,6 +342,9 @@ Table with terms used in the specification.
 | **Embedded objects** | An object embedded in the file/package i.e. an image, another spreadsheet or 3D object. |
 | **External cell references** | External cell references is a type of cell formula, which fetches data from another spreadsheet. |
 | **External objects** | An object that is linked as an OLE object, but is not embedded into the file/package. |
+| **File format** | A file format defines the data structure, scope of content and encoding of bits for any type of file. The file format may be standardised, it may contain other formats and it may be binary or text-based. The purpose of file format validation is two-fold: Checking if the file's purported format is correctly encoded and checking if any data quality rules, which delimits the file format's scope, are violated. |
+| **File format extension** | A file format extension is an identifier set at the end of the filename and delimited by a dot/period e.g. ".ods" or ".xlsx". The identifier helps the operating system to associate an application for opening the file. The file format extension is not authoritative, since the extension may be misleading if the file's format and the file's extension does not match. Instead a file format signature should be used. |
```

Comment:
I would leave the definition of extension at this:
A file format extension is an identifier set at the end of the filename, delimited by a dot/period, e.g. ".ods" or ".xlsx". The identifier helps the operating system associate an application to parse the file.

This part belongs elsewhere as text in the specification explaining why we believe that it's important that the file format and actual MIME type are consistent:

The file format extension is not authoritative since it may be arbitrarily assigned so that the file's format and extension aren't consistent. Instead, a file's format should be determined by analysis, e.g. matching via file format signature.

Reply:
I can adapt.